lib/chore/new_relic.rb

```diff
@@ -1,4 +1,4 @@
-gem 'newrelic_rpm', '>= 3.7.0'
+gem 'newrelic_rpm', '>= 3.15.0'
```

Comment:
It would be awesome to put this in the gemspec... I can't think of any good reason it lives here.